src/pages/_id/index.vue

```diff
@@ -245,6 +248,19 @@ export default {
     userAvatar() {
       return this.userInfo && this.userInfo.avatar;
     },
+    sortedOwnedNFTClassId() {
```

Comment:
I think can move this sort func to util, then my dashboard page can reuse later

Reply:
How about migrating this function to `/mixin/portfolio` in the `my-dashboard` when both branches are merged? Since we need to call `getNFTClassMetadataById()` to get the metadata, it's a bit inconvenient to do so in util (have to `import { store }` and call `store.getter.getNFTClassMetadataById`?).

Reply:
How about a getter function in store?

Reply:
Can do, thanks!!